sdk/tests/resources/tex-input-validation.js

```diff
@@ -260,12 +295,6 @@ var testCases = [
     expectedError: gl.INVALID_ENUM },
   { target: gl.TEXTURE_2D,
     internalFormat: gl.RGBA,
-    border: 1,
```

Comment:
Upon further thought, please leave this test case in. The OpenGL compatibility profile allows border to be 1, so there should be at least one test in this file ensuring that WebGL doesn't allow that to be passed.